src/views/Upload.vue

```diff
@@ -39,27 +52,87 @@ import UploadTab from '@/components/Upload/UploadTab.vue';
 import UploadFile from '@/components/Upload/UploadFile.vue';
 import UploadText from '@/components/Upload/UploadText.vue';
 
+import axios from 'axios';
+
 export default {
 	components: {
 		Navigation,
 		UploadTab,
 		UploadFile,
-		UploadText,
+		UploadText,	
 	},
 	data: function () {
 		return {
 			selectedTab: '',
 			tabs: ['Nahrať súbory', 'Vložiť text'],
+			files: [],
+			text: '',
 		};
 	},
+	methods: {	
+		handleFiles(filesArray){
+			Object.values(filesArray).forEach(file => {
```

Comment:
Too much complicated method.

`this.files = [...this.files, ...filesArray]`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment